scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/tpch/Q01.scala

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Did we make any changes to the files?

Reply:
no change for Q01-Q22.
modified TPCHQuery and TPCHSchemaProvider

Reply:
> no change for Q01-Q22. modified TPCHQuery and TPCHSchemaProvider

If there no change, you need to add "This file is copied from ..." at the beginning

If the file is changed, you may refer to an example at https://github.com/intel-analytics/BigDL/blob/main/python/orca/example/learn/tf2/yolov3/__init__.py

Reply:
Our scala style check only allow BigDL license in the header.  So maybe we split the original license to second part? Or we change our scala style check not contain ending "*/" in the header?

Reply:
just exclude these files in style check

Reply:
ok